lib/lapis_lazuli/generators/cucumber/template/features/helpers/user_helper.rb

```diff
@@ -0,0 +1,68 @@
+module User
+  extend LapisLazuli
+
+  class << self
+    @@data = nil
+
+    def load_user_data(user)
+      data = config('users.default-user')
+      begin
+        specific_data = config("users.#{user}")
+      rescue
```

Comment:
No error handling if the user doesnt exist. In that case you get the error:
 Unknown configuration variable 'users.production.tesdsst-user' and no default given! (RuntimeError)
      ./features/helpers/user_helper.rb:12:in `rescue in load_user_data'
 
  Expected:
Clearer feedback that user is not found in the config file?